mondrian/src/main/java/mondrian/spi/Dialect.java

```diff
@@ -161,6 +161,18 @@ String quoteIdentifier(
         String name);
 
     /**
+     * Encloses an identifier in quotation marks appropriate for the
+     * current SQL dialect. For example, in Oracle, where the identifiers
+     * are quoted using double-quotes,
+     * <code>quoteIdentifier("schema.name")</code> yields a string
+     * containing <code>"schema"."name"</code>.
+
+     * @param name Name to be quoted.
+     * @return Quoted identifier
+     */
+    String quoteIdentifierWithInnerDots(String name);
```

Comment:
I cannot do that without changing the method signature.

Are you sure that I need to do that?